packages/std/src/traits.rs

```diff
@@ -225,6 +225,13 @@ impl<'a, C: CustomQuery> QuerierWrapper<'a, C> {
         }
     }
 
+    pub fn to_empty(self) -> QuerierWrapper<'a, Empty> {
```

Comment:
I think this should be `into_*` as it consumes `self`.

Reply:
Yes, I think `into_empty` is good. Given the return type, it should be pretty self-explanatory and is not too long.
Doc comment would be good, though.